LICENSE

```diff
@@ -1,9 +1,9 @@
 MIT License
 
-Copyright (c) oemof developer group
+Copyright (c) 2015-2021, oemof developer group
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+The above copyright notice and this permission notice (including the next paragraph) shall be included in all copies or substantial portions of the Software.
```

Comment:
Whoever wants to give sub-licenses might also give a warranty. So the next paragraph should not be obligatory.